docs/advanced_examples/ReggresorComparison.ipynb

```diff
@@ -0,0 +1,514 @@
+{
+ "cells": [
+  {
```

Comment:
Hey. The filename "docs/advanced_examples/ReggresorComparison.ipynb" has typos. That's "regressor", with a single "g" and two "s".

Reply:
Inatention error, it's rectified thank you